ci/common.rb

```diff
@@ -38,6 +38,26 @@ def install_requirements(req_file, pip_options = nil, output = nil, use_venv = n
   end
 end
 
+def can_skip?
+  return false, [] if Gem.win_platform?
+
+  modified_checks = []
+  `git diff-tree --no-commit-id --name-only -r FETCH_HEAD origin/master`.each_line do |filename|
```

Comment:
You're right, it would be better, but since in our case we either run our tests against master or a release branch. For master, it's already in the source code, and I think that comparing 5.{y}.x against master will trigger all builds (which is what we want on a release branch).

So yeah, something to do when we have more time! 😄